### PR TITLE
Aib/0.6.0/migrate influencers analysis

### PR DESCRIPTION
**Pull request by arielibaba**

### CODE REVIEW

**Comment by bloussou** [comment]
The targeted branch must be `0.6.0` not `master`

File: notebooks/influencers-analysis/notebooks/main.ipynb [hunk 1]
```diff
@@ -920,7 +920,7 @@
    "source": [
     "## Atoti cube creation  \n",
     "\n",
-    "We have a simple cube that holds a single data store. Let's call it influencer_store."
+    "We have a simple cube that holds a single dinfluencer_tableata store. Let's call it ."
```
Comment (bloussou) [inline notebooks/influencers-analysis/notebooks/main.ipynb:923]: Typo

File: notebooks/influencers-analysis/notebooks/main.ipynb [hunk 2]
```diff
@@ -930,34 +930,14 @@
    "outputs": [],
    "source": [
     "# a session has to be created for atoti\n",
-    "config = tt.config.create_config(sampling_mode=tt.sampling.FULL)\n",
-    "session = tt.create_session(config=config)"
+    "# config = tt.config.create_config(sampling_mode=tt.sampling.FULL)\n",
```
Comment (bloussou) [inline notebooks/influencers-analysis/notebooks/main.ipynb:933]: This must be removed not commented

File: notebooks/influencers-analysis/notebooks/main.ipynb [hunk 3]
```diff
@@ -920,7 +920,7 @@
    "source": [
     "## Atoti cube creation  \n",
     "\n",
-    "We have a simple cube that holds a single data store. Let's call it influencer_store."
+    "We have a simple cube that holds a single data store. Let's call it influencer_table."
```
Comment (Skyscimitar) [inline notebooks/influencers-analysis/notebooks/main.ipynb:923]:
```suggestion
    "We have a simple cube that holds a single data table. Let's call it influencer_table."
```

File: notebooks/influencers-analysis/notebooks/main.ipynb [hunk 4]
```diff
@@ -1494,7 +1329,7 @@
     "- The average number of retweets and quotes for each tweet\n",
     "- The average number of likes for each tweet\n",
     "\n",
-    "We can observe that, in general, it's not those that produce the most tweets that are the most replied, retweeted or liked.\n",
+    "We can observe that, in general, it's not those who produce the most tweets that are the most replied, retweeted or liked.\n",
```
Comment (Skyscimitar) [inline notebooks/influencers-analysis/notebooks/main.ipynb:1332]:
```suggestion
    "We can observe that, in general, it's not those who produce the most tweets that are the most replied to, retweeted or liked.\n",
```

**Review by Skyscimitar — APPROVED** [review]
Just an occurrence of `store` to remove and a slight improvement in phrasing to fix before merging.

**Pull request merged**